### PR TITLE
Update collection ids to be accepted collections, inside the centrapa…

### PR DESCRIPTION
**Pull request by daallen303**
https://www.notion.so/centrapay/Return-Redeemable-Line-Items-d3cea51a658c4bc0a3248eed28e35dd6?pvs=4

### CODE REVIEW

File: legacy/src/api/payment-requests/payment-requests.md [hunk 1]
```diff
@@ -103,19 +103,30 @@ version (documented on this page) and the "legacy" version (documented at
...
+| acceptedCollections  | Array  | [Accepted Collections](###accepted-collection) for the payment request, when the "assetType" is `centrapay.token.*`|
 
 ★  For payment options which specify an address, there's a requirement to make a transaction on an external ledger.
 Once you have made that payment, you can use the transaction id to [Pay a Payment Request](#pay) using the legacy payment API.
 
 <a name="payment-condition">
 
+### Accepted Collection
+
+Payment Requests that have `centrapay.token.*` as a [Payment Option](###payment-option) will also return an array of Accepted Collections. An Accepted Collection can be used to determine if a [Centrapay Token][] is accepted for a particular Payment Request, and what line items the token can be used to purchase.
```
Comment (laurencemcknight) [inline legacy/src/api/payment-requests/payment-requests.md:121]: Will Accepted Collections be returned on the Payment Option or the Payment Request? The first sentence reads as if the Accepted Collections is returned on the Payment Request.

Comment (laurencemcknight) [inline legacy/src/api/payment-requests/payment-requests.md:121]:
```suggestion
Payment Requests that have `centrapay.token.*` as a [Payment Option](###payment-option) will also return an array of Accepted Collections. An Accepted Collection can be used to determine if a [Centrapay Token][] can be used to pay a Payment Request, and the line items able to be purchased using the token.
```

Comment (daallen303) [reply legacy/src/api/payment-requests/payment-requests.md:121]: It will be returned on the Payment Option, I will update it to be more clear. :)

Comment (daallen303) [reply legacy/src/api/payment-requests/payment-requests.md]: I have updated this, take a look and let me know what you think. :)

File: legacy/src/api/payment-requests/payment-requests.md [hunk 2]
```diff
@@ -103,19 +103,30 @@ version (documented on this page) and the "legacy" version (documented at
...
+| acceptedCollections  | Array  | [Accepted Collections](###accepted-collection) for the payment request, when the "assetType" is `centrapay.token.*`|
 
 ★  For payment options which specify an address, there's a requirement to make a transaction on an external ledger.
 Once you have made that payment, you can use the transaction id to [Pay a Payment Request](#pay) using the legacy payment API.
 
 <a name="payment-condition">
 
+### Accepted Collection
+
+If a Payment Request contains a `centrapay.token.*` [Payment Option](###payment-option) an array of Accepted Collections will be present inside the `centrapay.token` Payment Option. The Accepted Collections returned can be used to determine if a [Centrapay Token][] can be used to pay a Payment Request, and the line items able to be purchased using the token.
```
Comment (IsaacMurtagh) [inline legacy/src/api/payment-requests/payment-requests.md:121]: is [Centrapay Token][] correct? are you meaning to link it somewhere?

**Review by IsaacMurtagh — APPROVED** [review]

File: legacy/src/api/payment-requests/payment-requests.md [hunk 3]
```diff
@@ -103,19 +103,30 @@ version (documented on this page) and the "legacy" version (documented at
...
-| collectionIds  | Array  | Supported collectionIds for the payment request, when the "assetType" is `centrapay.token.*` |
+|     Field            |  Type  |                                         Description                                                                 |
+| -------------------- | ------ | ------------------------------------------------------------------------------------------------------------------- |
+| bitcoinAddress       | String | ★  Address to send Bitcoin, when the "assetType" is `bitcoin.*`                                                     |
+| cennzAddress         | String | ★  Address to send CPay, when the "assetType" is `cennznet.*`                                                       |
+| wavesAddress         | String | ★  Waves address to send Zap tokens, when the "assetType" is `zap.*`                                                |
+| productCodes         | Array  | Supported product codes for the payment request, when the "assetType" is `epay.nzd.*`                               |
+| acceptedCollections  | Array  | [Accepted Collections](###accepted-collection) for the payment request, when the "assetType" is `centrapay.token.*`|
```
Comment (laurencemcknight) [inline legacy/src/api/payment-requests/payment-requests.md:112]: This link does not work when I run the site locally

Comment (daallen303) [reply legacy/src/api/payment-requests/payment-requests.md]: This has been fixed :)

**Pull request merged**